browser/js/checkout/addressForm.html

```diff
@@ -0,0 +1,34 @@
+<form name="shippingForm">
+	<div class="form">
+		<div class="form-group col-xs-6">
+			<input class="form-control" ng-model="currentState.form.firstName" id="firstName" placeholder="First Name" required>
```

Comment:
@karn09 so I think what may be going on here is that the placeholder value will always show up, even if the ng-model has a value on the scope (except when a user is typing). So we may need some functionality to show ng-model OR the placeholder. I don't know how to do that, though... I think the cause is that placeholder is pure html and takes precedence over the angular jazz.